ramls/rtypes/post-json-201.raml

```diff
@@ -11,7 +11,7 @@ post:
         strict: false
         value: <<requestExample>>
   responses:
-    200:
+    201:
```

Comment:
it looks like you renamed this file, but both the new and original are still referenced in order.raml

Reply:
scratch that... I guess I don't understand the github diff notation.  I thought the -> in the filename indicated a rename

Reply:
It may looks a bit confused. I modified obsolete and unnecessary post-json-body-200.raml to my purposes (for response code 201) with renaming instead deletion  post-json-body-200.raml and creating new post-json-201.raml.

Reply:
I see what happened... I failed to notice the file name... `post-json-body-200.raml`, and mistook it for `post-json-200.raml`